### PR TITLE
Improvement/cx 942 revise readme

### PR DESCRIPTION
**Pull request by seewah**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://165-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md [hunk 1]
```diff
@@ -71,9 +95,9 @@ The library is **Browser only**, it does not support the **Node Context**.
  **[Webpack Sample App repository here.](https://github.com/onfido/onfido-sdk-web-sample-app/tree/0.0.2)**
```
Comment (dawiddylowicz) [inline README.md]: This link looks to be outdated, I believe it should point the latest state of repository, that is `https://github.com/onfido/onfido-sdk-web-sample-app/`

File: README.md [hunk 2]
```diff
@@ -88,92 +112,72 @@ verification component will be mounted. -->
...
-//replace the jwt token
-onfidoOut.setOptions({ token:"new token" });
-...
-//Open the modal
-onfidoOut.setOptions({ isModalOpen:true });
-```
+## JSON Web Token authentication
 
-The new options will be shallowly merged with the previous one. So one can pass only the differences to a get a new flow.
+Clients are authenticated using JSON Web Tokens (JWTs). The tokens are one use only and expire after 30 minutes. See [here](https://jwt.io/) for details on how JWTs work.
```
Comment (hjheath) [inline README.md]: Ricardo rewrote this whole section - we no longer allow customers to generate their own JWTs. Make sure you use his latest changes.

File: README.md [hunk 3]
```diff
@@ -4,27 +4,51 @@
...
+* [Customising SDK](#customising-sdk)
+* [Uploading files to Onfido API](#uploading-files-to-onfido-api)
+* [Creating checks](#creating-checks)
+* [Going live](#going-live)
 
-This is a plug-and-play SDK that leverages the Onfido SDK core, helping users take document and face captures that can then be sent to our backend APIs.
+## Overview
 
-All document captures are sent over WebSockets to our image evaluation API, to ensure your users are submitting a document image of adequate quality.
+This SDK provides a set of [React](https://facebook.github.io/react/) components for JavaScript applications to allow capturing of identity documents and face photos for the purpose of identity verification. The SDK offers a number of benefits to help you create the best onboarding / identity verification experience for your customers:
```
Comment (hjheath) [inline README.md]: React is an implementation detail - I don't think it's relevant to someone integrating with the SDK

File: README.md [hunk 4]
```diff
@@ -4,27 +4,51 @@
...
 
-To initialise the SDK, a connection to our WebSocket endpoint is required. Connections are authorized using [JWTs](https://jwt.io/), which can be generated on your server, or fetched from our JWT endpoint. Read about how to do this in the [authentication section](#authentication) below.
+Note: the SDK is only responsible for capturing photos. You still need to access the [Onfido API](https://documentation.onfido.com/) to upload photos, manage applicants and checks.
 
-## Screenshots
+All document captures are collected through browser file upload. On handheld devices the SDK uses the `accept="image/*"` attribute to give the option to take a photo using the native capture methods. There is a feature currently in beta that uses the [getUserMedia API](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) (where supported) to automatically detect and capture documents via a user’s webcam. This feature can be enabled using the [useWebcam](#public-options-and-methods) option. Face capture uses the webcam by default if one is available.
```
Comment (hjheath) [inline README.md]: I think there is too much technical detail about letting mobile users take photos. Perhaps: "Users are prompted to do upload a file containing an image of their document. On handheld devices they can also use the native camera to take a photo of their document."

I would not mention autocapture at all - we do not want people using it.

Comment (stefaniacardenas) [inline README.md]: I remember that during the Integration Experience Design meeting we mentioned that this paragraph was a bit too technical to be part of the Overview. Might be worth moving it to #customising-sdk one?

File: README.md [hunk 5]
```diff
@@ -4,27 +4,51 @@
...
-## Example
+## Getting started
+
+### 1. Obtaining an API token
+
+In order to start integration, you will need the **API token**. You can use our [sandbox](https://documentation.onfido.com/#testing) environment to test your integration, and you will find the sandbox token inside your [Onfido Dashboard](https://onfido.com/dashboard/api/tokens).
+
+### 2. Generating a JSON Web Token
+
+For security reasons, instead of using the API token directly in you client-side code, you will need to supply a short-lived JSON Web Token (JWT) to initialise the SDK. The easiest way to generate a JWT is via the [JWT endpoint](https://documentation.onfido.com/#json-web-tokens) in the Onfido API:
```
Comment (hjheath) [inline README.md]: Make clear that you need a new token every time you initialise the SDK.

File: README.md [hunk 6]
```diff
@@ -4,27 +4,51 @@
...
+## Overview
 
-All document captures are sent over WebSockets to our image evaluation API, to ensure your users are submitting a document image of adequate quality.
+This SDK provides a set of [React](https://facebook.github.io/react/) components for JavaScript applications to allow capturing of identity documents and face photos for the purpose of identity verification. The SDK offers a number of benefits to help you create the best onboarding / identity verification experience for your customers:
 
-All document captures are collected through file upload. On handheld devices the SDK uses the `accept="image/*"` attribute to give the option to take a photo using the native capture methods. There is a feature currently in beta that uses the [getUserMedia API](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia) (where supported) to automatically detect and capture documents via a user’s webcam. This feature can be enabled using the [useWebcam](#public-options-and-methods) option. Face capture uses the webcam by default if one is available.
+- Carefully designed UI to guide your customers through the entire photo-capturing process
```
Comment (hjheath) [inline README.md]: I really like this section 👍

File: README.md [hunk 7]
```diff
@@ -88,92 +112,72 @@ verification component will be mounted. -->
...
-  },
   onComplete: function(capturesHash) {
-    // callback for when everything is complete
-  },
-  steps: [
-    //you can customize the flow of different steps of the SDK flow
-    //you can change the order or remove steps
-    'welcome','document','face','complete'
-  ]
+    console.log('Photos captured');
```
Comment (hjheath) [inline README.md]: This seems weird. In the development app we do: 

```
console.log("everything is complete")
console.log(Onfido.getCaptures())
```

File: README.md [hunk 8]
```diff
@@ -189,7 +193,88 @@ A breakdown of the options and methods available to the SDK.
 
 - **`onComplete {Function} optional`**
 
-  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full API requests will be made.
+  Callback that fires when both the document and face have successfully been captured. It returns an object that contains both captures. This event data should sent to your backend where the full [API requests](#uploading-files-to-onfido-api) will be made.
+
+  Here is an `onComplete` callback example, making use of the `getCaptures` convenience function:
```
Comment (hjheath) [inline README.md]: We should fully document the other public methods of the API:

- tearDown
- getCaptures

File: README.md (hunk 3 above)
Comment (stefaniacardenas) [inline README.md]: From the first few lines it is not clear that this SDK is used to capture images that will be consumed exclusively by the Onfido API and it's not just a capturing tool that you can use with any service.
Maybe we could write something like
```This SDK provides a set of [React](https://facebook.github.io/react/) components for JavaScript applications to allow capturing of identity documents and face photos for the purpose of identity verification performed by the Onfido backend APIs.```

File: README.md [hunk 9]
```diff
@@ -88,92 +112,72 @@ verification component will be mounted. -->
...
+You need a new JWT each time you initialise the SDK. You can obtain a JWT in two ways:
 
-A breakdown of the options and methods available to the SDK.
+### 1. Through Onfido API
 
-- **`token {String} required`**
+The Onfido [API](https://documentation.onfido.com/) exposes a JWT endpoint. See the API [documentation](https://documentation.onfido.com/#json-web-tokens) for details.
 
-  A JWT is required in order to authorise with our WebSocket endpoint. If one isn’t present, an exception will be thrown.
+### 2. Generating your own
```
Comment (stefaniacardenas) [inline README.md]: Given that we want to deprecate this maybe we should remove it from the documentation

**Comment by rfreitas** [comment]
Travis automatic deployment: https://165-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://165-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by hjheath** [comment]
Can you add a bit on the `tearDown` method? Otherwise looks good to me

**Comment by rfreitas** [comment]
Travis automatic deployment: https://165-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://165-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**